### DISCUSSION TITLE
WaveSurfer was not working properly

### DISCUSSION THREAD

**Discussion by Diwanshumidha** [community/community#68538] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

WaveSurfer was not working properly if we are fetching video from api or external route 

steps to reproduce:
1. Just make a react app with surfer js
2. Try to load using external video link

**Comment by MacMillon-VS** [comment]
Make sure you are fetching it from https. If we are using local mp3 file it will work

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
